### PR TITLE
added alert feature

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: Documentation/Code Snippets/code.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    overflow-x: auto;
+    font-size: 1rem;
+    margin: 2rem 0;
+  }
+  .code{
+      font-size: 1.5rem;
+  }
+  .language{
+      color: white;
+      background-color: #1f4287;
```
Comment (harshitadaswani) [inline Documentation/Code Snippets/code.css]: use color variable

File: Documentation/Code Snippets/code.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      width: fit-content;
+  } 
+  
+   */
+  .code-wrapper pre{
+      background-color: #071e3d;
+      color: white;
+      padding: .8rem 2rem;
+      border-radius: .5rem;
+      border: .2px solid white;
```
Comment (harshitadaswani) [inline Documentation/Code Snippets/code.css]: use 2px

File: Documentation/documentation.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+.sidebar ul li a {
+  text-decoration: none;
+  padding: 2rem;
+  font-size: 1.5rem;
+}
+
+.main--content {
+  /* border: 2px solid green; */
+  width: 109rem;
```
Comment (harshitadaswani) [inline Documentation/documentation.css]: use percentage value instead of such large values

File: components/Alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Documentation/documentation.css">
+    <link rel="stylesheet" href="/stylesheets/utility.css">
+    <link rel="stylesheet" href="/components/Alert/alert.css">
+    <title>Alert</title>
+</head>
+<body>
+   
+    <nav class="navigation">
+        <!-- <img class="super-man" src="/images/flame-man.svg" alt=""> -->
```
Comment (harshitadaswani) [inline components/Alert/alert.html]: remove comment

**Review by harshitadaswani — CHANGES_REQUESTED** [review]
Refactor suggested changes

**Comment by ajit-sharmaGH** [comment]
Code looks clean

**Review by harshitadaswani — APPROVED** [review]
The code looks clean. You can go ahead

**Pull request merged**